doc/experiment.rst

```diff
@@ -1,326 +1,438 @@
 
-  model = exp.create_model("hello_world", srun)
-  exp.start(model, block=True, summary=True)
+  .. code-block:: python
 
-  print(exp.get_status(model))
+      exp = Experiment("name-of-experiment", launcher="slurm")  # slurm launcher
 
-The above will run ``srun -n 32 -N 1 echo Hello World!``, monitor its
-execution, and inform the user when it is completed. This driver script can be
-executed in an interactive allocation, or placed into a batch script as follows:
+  To instruct SmartSim to search for a `launcher` on your machine, set the `launcher`
+  argument to `"auto"` during ``Experiment`` initialization as shown below:
 
-.. code-block:: bash
+  .. code-block:: python
 
-    #!/bin/bash
-    #SBATCH --exclusive
-    #SBATCH --nodes=1
-    #SBATCH --ntasks-per-node=32
-    #SBATCH --time=00:10:00
+      exp = Experiment("name-of-experiment", launcher="auto")  # auto-detect launcher
 
-    python /path/to/script.py
+  If the systems `launcher` cannot be found or no `launcher` argument is provided, the default value of
+  `launcher="local"` will be used.
 
+.. _entities_exp_docs:
+========
+Entities
+========
+Entities are SmartSim API objects that can be launched and
+managed on the compute system via the Experiment API. While the
+``Experiment`` object is intended to be instantiated once in a
+SmartSim driver script, there is no limit to the number of SmartSim entities
+within an ``Experiment``. In the following subsections, we define the
+general purpose of the three entities that can be created via
+Experiment API factory methods:
+
+* ``Orchestrator``
+* ``Model``
+* ``Ensemble``
+
+To create a reference to a newly instantiated entity object, use the associated
+``Experiment.create_...()`` function.
+
+.. list-table:: Experiment API Entity Creation
+   :widths: 20 65 25
+   :header-rows: 1
+
+   * - Factory Method
+     - Example
+     - Return Type
+   * - ``create_database()``
+     - ``orch = exp.create_database([port, db_nodes, ...])``
+     - :ref:`Orchestrator <orchestrator_api>`
+   * - ``create_model()``
+     - ``model = exp.create_model(name, run_settings)``
+     - :ref:`Model <model_api>`
+   * - ``create_ensemble()``
+     - ``ensemble = exp.create_ensemble(name[, params, ...])``
+     - :ref:`Ensemble <ensemble_api>`
+
+After initialization via the ``Experiment`` factory methods, each entity can be started, monitored, and stopped
+using the :ref:`Experiment API<experiment_api>`.
+
+.. list-table:: Interact with Entities during the Experiment
+   :widths: 25 55 25
+   :header-rows: 1
+
+   * - Experiment Function
+     - Example
+     - Desc
+   * - ``start()``
+     - ``exp.start(*args[, block, summary, ...])``
+     - Launch an Entity
+   * - ``stop()``
+     - ``exp.stop(*args)``
+     - Stop an Entity
+   * - ``get_status()``
+     - ``exp.get_status(*args)``
+     - Retrieve Entity Status
+
+.. _orchestrator_exp_docs:
+Orchestrator
```

Comment:
For this section, I think it's sufficent to say that the `create_database` method allows users to create standalone orchestrators. Mention that colocated orchestrators are actually requested using the model API not the experiment API. Then also refer to the standalone vs colocated distinction in the documentation on Model